libraries/AP_HAL_ChibiOS/hwdef/uav-dev-fc-um982/README.md

```diff
@@ -0,0 +1,149 @@
+# UAV-DEV-FC-UM982 Flight Controller with RTK GNSS and GNSS Heading capability via dual-antenna setup
+
+The UAV-DEV-FC-UM92 Flight Controller is sold by [UAV-DEV GmbH Webshop](https://shop.uav-dev.com/)
+
+## Features
+#### Processor
+- STM32H743
+- 480MHz
+- 2MB Flash
+- 1MB RAM
+#### Sensors
+- IMU TDK Invensense ICM-45686
+- Barometer Infineon DPS310
+- Magnetometer Bosch BMM150
+- GNSS Unicore UM982 L1/L2/L5 RTK GNSS with GNSS Heading
+#### Dimensions
+- Size: 50mm x 50mm (without SMA connector) x 15mm
+- Weight: 22g with microSD card
+#### Power
+- 5.2V input via JST-GH
+#### Interfaces
+- **USB-C**
+- **Ethernet**
+  - 100 MBit/s
+- **PWM**
+  - 6 x PWM / DShot, 2 x Bidirectional-DShot capable
+- **RC Input**
+  - UART / RCININT multi protocol
```

Comment:
I am confused about RC. You have a CPU pin defined for timer interrupt RC input for  and a UART6 protocol set for RCin, but I don't see two separate pins for this. You cannot have  them  tied together, or am I missing something?

Reply:
I am now only use UART6 for RCIN